CHANGELOG.md

```diff
@@ -2,15 +2,17 @@
 
 ## Unreleased
 
+### Fixes
+
+- Android: Fix for SIGSEGV reported by Sentry when a NullReferenceException is thrown in Release mode ([#3694](https://github.com/getsentry/sentry-dotnet/pull/3694))
+  - To turn it on, set: `options.HandlerStrategy = NdkHandlerStrategy.SENTRY_HANDLER_STRATEGY_CHAIN_AT_START;`
```

Comment:
```suggestion
```

This isn't an option right? We always use `SENTRY_HANDLER_STRATEGY_CHAIN_AT_START` when initialising the integration from the .NET SDK?

Reply:
It isn't anymore indeed, I forgot to update it. thanks for noticing this